public/app/features/canvas/runtime/element.tsx

```diff
@@ -486,6 +486,24 @@ export class ElementState implements LayerElement {
   };
 
   onElementClick = (event: React.MouseEvent) => {
+    this.onTooltipCallback();
+  };
+
+  onElementKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Enter') {
+      const scene = this.getScene();
+      if (scene?.tooltipCallback && scene.tooltip?.anchorPoint) {
+        const rect = this.div?.getBoundingClientRect();
+        scene.tooltipCallback({
+          anchorPoint: { x: rect?.right ?? scene.tooltip.anchorPoint.x, y: rect?.top ?? scene.tooltip.anchorPoint.y },
+          element: this,
+          isOpen: true,
+        });
+      }
+    }
+  };
```

Comment:
maybe something like?

```suggestion
onElementKeyDown = (event: React.KeyboardEvent) => {
    if (event.key === 'Enter' && (event.currentTarget instanceof HTMLElement || event.currentTarget instanceof SVGElement)) {
      const scene = this.getScene();
      scene?.select({targets: [event.currentTarget]})
    }
  };
```

Reply:
Thank you for the types solution! It works! 💙
Unfortunately, I didn't find a more elegant solution here 🙁